ci/lepton/model_convergence/configs/recipes/esm2_accelerate.yaml

```diff
@@ -0,0 +1,37 @@
+# @package _global_
+defaults:
+  - /base
+  - _self_
+
+resource_shape: gpu.2xh200
+
+# Base configuration
+recipe_subdir: esm2_accelerate
+train_cmnd: train
```

Comment:
i think `cmd` is probably the more standard abbreviation for 'command'?

Reply:
updated